backends/arm/test/ops/test_slice.py

```diff
@@ -0,0 +1,116 @@
+# Copyright 2024 Arm Limited and/or its affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import unittest
+from typing import Tuple
+
+import torch
+from executorch.backends.arm.quantizer.arm_quantizer import (
+    ArmQuantizer,
+    get_symmetric_quantization_config,
+)
+from executorch.backends.arm.test import common
+from executorch.backends.arm.test.tester.arm_tester import ArmTester
+from executorch.backends.xnnpack.test.tester.tester import Quantize
+from parameterized import parameterized
+
+
+class TestSimpleSlice(unittest.TestCase):
+
+    class Slice(torch.nn.Module):
+
+        sizes = [(10), (10, 10), (10, 10, 10), ((1, 12, 10, 10))]
+        test_tensors = [(torch.ones(n),) for n in sizes]
+
+        def forward(self, x: torch.Tensor):
+            if x.dim() == 1:
+                return x[3:-3]
+            elif x.dim() == 2:
+                return x[1:3, 3:5]
+            elif x.dim() == 3:
+                return x[0:7, 0:1, 0:8]
+            elif x.dim() == 4:
+                return x[:, 2:5, 3:5, 4:5]
+
+    def _test_slice_tosa_MI_pipeline(
+        self, module: torch.nn.Module, test_data: torch.Tensor
+    ):
+        (
+            ArmTester(
+                module,
+                example_inputs=test_data,
+                compile_spec=common.get_tosa_compile_spec(),
+            )
+            .export()
+            .check(["torch.ops.aten.slice.Tensor"])
```

Comment:
Do you want to count slice ops too? I guess your test has variable ops so it might be tricky